### PR TITLE
ka-customizer

### PR DESCRIPTION
**Pull request by mattdangerw**
Little gui for playing with the new CLI options
https://phabricator.endlessm.com/T13878

### CODE REVIEW

**Comment by nedrichards** [comment]
This dir/tool should have a README to explain how to install/run it, especially given the intended audience!

File: tools/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+-------------
+KA Custimizer is a small GUI tool to tweak modules and themes of a knowledge
+app.
+
+#### Installing
+To install, run open a terminal and copy and enter the following command.
+```
+bash -exc "$(wget https://raw.githubusercontent.com/endlessm/eos-knowledge-lib/master/tools/ka-customizer-installer -O -)"
+```
+Installing may take a while! The bleeding edge versions of our flatpak runtimes
```
Comment (ptomato) [inline tools/README.md]: drive-by comment: "bleeding edge" is kind of developer jargon...

File: tools/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Tools
+=====
+
+KA Customizer
+-------------
+KA Custimizer is a small GUI tool to tweak modules and themes of a knowledge
```
Comment (ptomato) [inline tools/README.md]: Typo - Customizer

File: tools/README.md (hunk 1 above)
Comment (mattdangerw) [inline tools/README.md]: That was my attempt to be less jargon-y then master branch. "Nightly unstable" perhaps?

**Comment by rmacqueen** [comment]
Tested it out. Worked for me but the delay from hitting run to actually launching the app was like 5 mins. Is that expected?

**Comment by mattdangerw** [comment]
@rmacqueen if the "update" switch was on, it will try and make sure you runtime, sdk and app are up to date before running. Sounds like you had some updates to your runtime/sdk which would take a while.

Wanted that switch on be default to avoid everyone testing on different versions of things. But if you toggle "update" off you should run right away.

File: tools/ka-customizer [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    halign: Gtk.Align.START,
+    show_default_item: true,
+});
+let selected_app_id = '';
+for (let app of Gio.app_info_get_all()) {
+    let app_id = app.get_id().split('.desktop')[0];
+    let match = app_id.match(/com\.endlessm\..+\.(.+)/);
+    if (match === null)
+        continue;
+    // Really bad heuristic for knowledge app detection. Check app id looks like
```
Comment (mattdangerw) [inline tools/ka-customizer]: Landed a better check here, see most recent commit.

File: tools/ka-customizer [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    show_default_item: true,
+});
+let selected_app_id = '';
+for (let app of Gio.app_info_get_all()) {
+    let app_id = app.get_id().split('.desktop')[0];
+    let match = app_id.match(/com\.endlessm\..+\.(.+)/);
+    if (match === null)
+        continue;
+    let locale = match[1];
+    // Check if knowledge app by looking for ekn directory
```
Comment (rmacqueen) [inline tools/ka-customizer]: nice!

**Comment by rmacqueen** [comment]
CR PASS
